docs/cmdline/create.md

```diff
@@ -0,0 +1,68 @@
+---
+title: terramate create - Command
+description: With the terramate create command you can create a new stack in the current project.
+
+prev:
+  text: 'Stacks'
+  link: '/stacks/'
+
+next:
+  text: 'Sharing Data'
+  link: '/data-sharing/'
+---
+
+# Create
+
+The `create` command creates a new stack. This command accepts a path that will
+be used to create the stack. The `create` command will create intermediate
+directories as required. Whenever the `create` command is used, the code generation will run automatically to ensure all code relevant for the newly created stack will be generated.
+
+## Usage
+
+`terramate create [options] PATH`
+
+## Examples
+
+Create a new stack:
+
+```bash
+terramate create path/to/stack
+```
+
+Complete Example:
+
+```bash
+terramate create path/to/stack \
+    --name foo-stack \
+    --description "This is an example" \
+    --tags app,prd \
+    --after ../../after-another-stack,../../after-this-stack \
+    --before ../../before-this-stack,../../before-another-stack \
+    --ignore-existing \
+    --no-generate
+```
+
+Initialize Terramate in an existing Terraform project:
+
+```bash
+terramate create --all-terraform
+```
+
+This is helpful when you want to onboard Terramate in an existing
+Terraform project. `--all-terraform` will create a Terramate configuration
+file in every Terraform directory that contains a `terraform.backend` block.
+
+
+## Options
+
+- `--id=STRING` ID of the stack. Defaults to a random UUIDv4 (Using the default is highly recommended).
+- `--name=STRING` Name of the stack. Defaults to stack dir base name.
+- `--description=STRING` Description of the stack. Defaults to the stack name.
+- `--tags=STRING` Adds tags to the stack. `--tags b --tags b` is the same as `--tags b,b`.
```

Comment:
```suggestion
- `--tags=STRING` Adds tags to the stack. Example: `--tags a --tags b.
```

there currently is a bug preventing `--tags a,b` also the text duplicates `b,b` instead of `a,b` ;) we can update the docs, once the bug is fixed.